torchaudio/pipelines/_wav2vec2/impl.py

```diff
@@ -70,11 +88,13 @@ def get_model(self, *, dl_kwargs=None) -> Wav2Vec2Model:
             dl_kwargs (dictionary of keyword arguments): Passed to :func:`torch.hub.load_state_dict_from_url`.
         """
         model_type = self._params.pop("model_type", None)
+        normalize_waveform = self._params.pop("normalize_waveform", None)
```

Comment:
Since this operation is performed on the module-level instance, let's use non-destructive accessor like `get`.